server/service/src/apis/permissions.rs

```diff
@@ -305,7 +304,6 @@ pub fn permission_mapping() -> HashMap<i16, Permissions> {
         (107, Permissions::DownloadBatchDocument),
         (108, Permissions::DeleteBatchDocument),
         (109, Permissions::AddStocktakeLines),
-        (110, Permissions::ViewStocktakeLines),
```

Comment:
These are the permission as they are coming from mSupply please don't remove them. If we use them at a later point it would be a pain to insert them back at the correct location...

Reply:
It can be removed since mSupply has an issue to remove ViewStocktakeLines permission https://github.com/openmsupply/msupply/pull/13725

Reply:
Ok, please add a comment why this array entry is unused (and what was there before) though...